test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionInvokerParameterBindingTest.cs

```diff
@@ -3,21 +3,21 @@
 
 using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 using System.Reflection;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc.Abstractions;
 using Microsoft.AspNetCore.Mvc.Controllers;
 using Microsoft.AspNetCore.Mvc.ModelBinding;
-using Microsoft.AspNetCore.Mvc.ModelBinding.Metadata;
 using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
 using Microsoft.AspNetCore.Routing;
 using Moq;
 using Xunit;
 
 namespace Microsoft.AspNetCore.Mvc.Internal
 {
-    public class DefaultControllerArgumentBinderTests
+    public class ControllerActionInvokerParameterBindingTest
```

Comment:
Using a new type since `ControllerActionInvokerTest` primarily focuses on action invocation now and is already pretty lengthy.